unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/MapGlobalUsersPositionMarkerController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MapGlobalUsersPositionMarkerController : IDisposable
```

Comment:
Could we add a documentation header here to explain what is the responsibility of this class? Thanks!